types.go

```diff
@@ -1,6 +1,6 @@
 package safecast
 
-// This files is highly inspired from https://pkg.go.dev/golang.org/x/exp/constraints
+// These files are highly inspired from https://pkg.go.dev/golang.org/x/exp/constraints
```

Comment:
afaik it's only here that the types are set, thus the original comment is almost correct (modulo files->file):
```suggestion
// This file is highly inspired from https://pkg.go.dev/golang.org/x/exp/constraints
```